Build/ProjectTypes/Test.props

```diff
@@ -13,7 +13,7 @@
   <ItemGroup>
     <PackageReference Include="JetBrains.Annotations" Version="2022.1.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.10.0" />
-    <PackageReference Include="Moq" Version="4.16.1" PrivateAssets="all" />
+    <PackageReference Include="Moq" Version="4.18.4" PrivateAssets="all" />
     <PackageReference Include="Moq.VerifiableSequence" Version="4.16.1.2" PrivateAssets="all" />
```

Comment:
There isn't a hard and fast rule yet. For consistency, having our Moq.VerifiableSequence build match Moq would be a nice touch, knowing the builds are officially compatible by also getting a `v4.18.4.1` version. please check with @a-ctor if that's a good practice to establish.

Reply:
I don't think is is necessary to keep your package version up-to-date if nothing has changed. We started with a matching version which kind of implies that we update the version but I don't think we need to. Its just unnecessary work as long as we don't have to fix anything.

If a higher version of Moq is installed it will be used for `Moq.VerifiableSequence` and in the case that something is broken our CI tests fail. If that happens we can update the package and bump the version to the one we are using right now. I see the chance of something silently breaking as very small since we mostly wrap Moq methods in the package.